### DISCUSSION TITLE
Remote docker deployment CORS error

### DISCUSSION THREAD

**Discussion by Aquariius** [reconmap/reconmap#70] [Q&A] [answered]
Hi, i am currently trying to deploy the docker application on a remote local server. So it's not localhost. When I am typing the user and password. I got a network error. When i go in the developer console i got a CORS error:
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://localhost:8080/users/login. (Reason: CORS request did not succeed)

i tried to change the allow address in the config.json api, but did not work. I think the error is on the frontend but can seem to find the problem. Can you help me ?

**Comment by santiagolizardo** [comment, marked as answer]
Hi @Aquariius ; Have you also modified the `environment.js` file to point to your API?
This line specifically: https://github.com/reconmap/reconmap/blob/master/environment.js#L5

**Reply by Aquariius** [reply]
Thx a lot this, and chagnin the cors in the api container made it work.

**Comment by psyray** [comment]
Hi, @santiagolizardo Thanks for this.
For those who have a 500 error while trying to login, remove the `\` before the `$`
@santiagolizardo I think you should remove it from the doc
![image](https://user-images.githubusercontent.com/1230954/150652192-f693e78c-11df-48e9-a4ae-86e290a31e55.png)

**Reply by santiagolizardo** [reply]
HI @psyray ! You are absolutely right. The dollar signs should not be escaped in that example. I'll get the docs fixed tonight, Thanks!

**Reply by santiagolizardo** [reply]
On further inspection, I can see that the escaping on that case is required because I am catting using [bash here docs](https://tldp.org/LDP/abs/html/here-docs.html). But in any case I'll make a note on the page for those who might be copying and pasting the content of that example without cat.

**Reply by psyray** [reply]
Yes you're right didn't figure this out because i've just copy paste the conf to put it in a directive snippets on ispconfig
